src/main/java/Duke.java

```diff
@@ -1,10 +1,137 @@
+import java.util.Scanner;
+
 public class Duke {
-    public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
+    public static class Task {
+        protected String description;
+        protected boolean isDone;
+        protected char taskType;
+
+        public Task(String description) {
+            this.description =  description;
+            this.isDone = false;
+        }
+
+        public String getStatusIcon() {
+            return (isDone ? "X" : " ");
+        }
+        public void markAsDone() {
+            this.isDone = true;
+        }
+
+    }
+
+    static String exitTrigger = "bye";
+    static String listTrigger = "list";
+    static String doneTrigger = "done";
```

Comment:
Constants name should be in all caps, perhaps you could change them to EXIT_TRIGGER?

Reply:
If I am not wrong, constant names should be in caps and underscore as a separator, so maybe a few changes can be made to those names?
```suggestion
    static String EXIT_TRIGGER= "bye";
    static String LIST_TRIGGER = "list";
    static String DONE_TRIGGER = "done";
```

Reply:
These string constants could be all caps with perhaps a final keyword in front to match coding standards.